includes/compat/subscriptions/class-wc-payment-gateway-wcpay-subscriptions-compat.php

```diff
@@ -46,6 +50,15 @@ public function __construct( ...$args ) {
 
 		// Display the credit card used for a subscription in the "My Subscriptions" table.
 		add_filter( 'woocommerce_my_subscriptions_payment_method', [ $this, 'maybe_render_subscription_payment_method' ], 10, 2 );
+
+		// Allow store managers to manually set Stripe as the payment method on a subscription.
+		add_filter( 'woocommerce_subscription_payment_meta', [ $this, 'add_subscription_payment_meta' ], 10, 2 );
+		add_filter( 'woocommerce_subscription_validate_payment_meta', [ $this, 'validate_subscription_payment_meta' ], 10, 3 );
+		add_action( 'wcs_save_other_payment_meta', [ $this, 'save_meta_in_order_tokens' ], 10, 4 );
+		// Enqueue JS hack when Subscriptions does not provide the meta input filter.
+		if ( version_compare( WC_Subscriptions::$version, '3.0.7', '<=' ) ) {
```

Comment:
Let's add a `TODO` [in the main class](https://github.com/Automattic/woocommerce-payments/blob/a2217e35de3ea3cc43354fdde8a03f398e94bad0/includes/class-wc-payments.php#L108) to remove the JS hack when we bump the minimum required Subscriptions version to 3.0.8 or higher.

Reply:
Good call. Added in https://github.com/Automattic/woocommerce-payments/pull/892/commits/5e3416bc51c67878dc0f0ce66e25d713ce070aa6.